src/helpers/registrars.ts

```diff
@@ -155,6 +164,37 @@ function buildContainerHandler(config: ContainerDirectiveConfig): ContainerDirec
     };
 }
 
+function buildCodeContainerHandler(
```

Comment:
nit:
maybe it would be helpful to add a note explaining when it is most convenient to use nameOrConfig.type equals 'code_block'